src/System.Runtime.WindowsRuntime/dir.props

```diff
@@ -4,6 +4,8 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.14.0</AssemblyVersion>
     <AssemblyKey>ECMA</AssemblyKey>
+    <IsNETCoreApp>true</IsNETCoreApp> 
+    <IsNETCoreAppRef>false</IsNETCoreAppRef> 
```

Comment:
Are you planning to add the package with the ref in another PR?

Reply:
Yes,  Do you have example for publishing individual package for reference assembly?

Reply:
or if it is very easy to publish the ref assembly in a individual package, then I can just add it into this PR.

Reply:
I don't think there is a exact example I suggest starting by bringing back the package for this library,https://github.com/dotnet/corefx/blob/release/1.1.0/src/System.Runtime.WindowsRuntime/pkg/System.Runtime.WindowsRuntime.pkgproj that is the last time we shipped this.

Reply:
Copied the one from https://github.com/ericstj/corefx/commit/9e5e4db1d50ae7b0d201249a81d0dd8e4d30cabf#diff-a8bb3eae1cb70b8353570e4e503e632d